climpred/toy.py

```diff
@@ -0,0 +1,259 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+from climpred.bootstrap import bootstrap_compute
+from climpred.prediction import compute_perfect_model
+from climpred.tutorial import load_dataset
+
+# standard setup
+lead = xr.DataArray(np.arange(0, 20, 1), dims='lead')
+lead['lead'] = lead.values
+
+control = load_dataset('MPI-control-1D')['tos'].isel(period=-1, area=1)
+# amplitudes of the signal and noise
+noise_amplitude = control.std().values * 2.5
```

Comment:
Maybe save control.std().values into a variable so users don't have to run the calc twice on import